pkgs/by-name/in/intel-media-driver/package.nix

```diff
@@ -11,13 +11,14 @@
   libdrm,
   enableX11 ? stdenv.hostPlatform.isLinux,
   libX11,
+  nix-update-script,
   # for passhtru.tests
   pkgsi686Linux,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "intel-media-driver";
-  version = "24.3.4";
+  version = "25.1.3";
```

Comment:
We only want to update to releases, so the latest version is https://github.com/intel/media-driver/releases/tag/intel-media-24.4.4